Node/src/TeamsChatConnector.ts

```diff
@@ -110,6 +112,17 @@ export class TeamsChatConnector extends builder.ChatConnector {
   }
 
   /**
+  * This method is to resolve regex or LUIS Bot being affected by at mentions
+  * Set true would have incoming messages stripped off source Bot at mentions
+  * The default would be false
```

Comment:
??? BotBuilderSDK has [several bits of middleware ](https://github.com/Microsoft/BotBuilder/blob/master/Node/core/src/middleware/Middleware.ts) defined. Also the middleware interface is built in to the Node BotBuilder SDK, it doesn't rely on express/restify.